src/main/java/uk/gov/pay/api/app/config/PublicApiModule.java

```diff
@@ -64,9 +66,26 @@ public ObjectMapper provideObjectMapper() {
 
     @Provides
     public RateLimiter provideRateLimiter() {
-        return new RateLimiter(configuration.getRateLimiterConfig().getRate(), 
+
+        LocalRateLimiter localRateLimiter=getLocalRateLimiter();
+        RedisRateLimiter redisRateLimiter=getRedisRateLimiter();
+
+        return new RateLimiter(configuration.getRateLimiterConfig().getRate(),
                 configuration.getRateLimiterConfig().getRateForPost(),
-                configuration.getRateLimiterConfig().getAuditRate(), 
-                configuration.getRateLimiterConfig().getPerMillis());
+                configuration.getRateLimiterConfig().getAuditRate(),
+                configuration.getRateLimiterConfig().getPerMillis(),
+                configuration.getRateLimiterConfig().getNoOfNodes(),
+                localRateLimiter,
+                getRedisRateLimiter());
```

Comment:
this should use the redisRateLimiter variable in line 71

Reply:
Updated